private/src/styles/blocks/core-blocks/_navigation.scss

```diff
@@ -0,0 +1,67 @@
+.wp-block-navigation {
+  padding: 20px;
+  min-width: 100%;
+}
+
+.wp-block-navigation .wp-block-navigation__container {
+  display: flex;
+  flex-direction: column;
+  height: 60px;
```

Comment:
this causes issues with longer menu items, try testing using the "toxic twitter" menu, and you'll hopefully see what i mean:
![](http://bigbite.im/i/FL4R1h+)

Reply:
resolved